src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/AmendProbationBookingService.kt

```diff
@@ -58,6 +58,11 @@ class AmendProbationBookingService(
       comments = request.comments
       this.amendedBy = amendedBy.username
       amendedTime = now()
+      notesForPrisoners = request.notesForPrisoners
+
+      if (amendedBy is ExternalUser) {
+        notesForStaff = request.notesForStaff
```

Comment:
We will only use private notes when requested by an external user.

Reply:
Is this right? It might be, but I though prison staff could enter private or public notes?

Reply:
🤦 I have it the wrong way round

Reply:
Not sure about that - notesForPrisoners can only be seen/written by prison staff, but notesForStaff can be seen and written by all?